### DISCUSSION TITLE
Engage the community

### DISCUSSION THREAD

**Discussion by ihsaan-ullah** [codalab/codabench#1332] [Ideas]
Ideas to engage more the community:
- We could embed the last topics from the Google-groups on the home page near the news.
- We could link this page as a "hall of contributors": https://github.com/codalab/codabench/graphs/contributors
- We could have a "hall of papers" based on Google scholar papers citing CodaLab / Codabench
For hall of papers: https://pypi.org/project/scholarly/ 
- There is also a Github feature for sponsoring (#901)
- Show that v1 bundle are compatible with Codabench, so no big entry barrier,  the platform is fast and smooth

Discussion started from #955

**Comment by Didayolo** [comment]
Hall of papers:

We could have the 5 last papers citing the platform shown on the home page, and a "show more" button redirecting to a page with all papers citing the platform.

This way we see the research work using the platform, give weight to the project, and incentivize researchers to cite the platform.

**Comment by Didayolo** [comment]
We started a Newsletter:
- https://docs.codabench.org/latest/Newsletters_Archive/CodaLab-in-2024/
- https://docs.codabench.org/latest/Newsletters_Archive/CodaLab-in-2025/
